src/main/kotlin/de/nycode/bankobot/twitch/WebhookServer.kt

```diff
@@ -48,12 +51,22 @@ internal fun Kord.launchServer() = embeddedServer(CIO) {
     install(Routing) {
         route("twitch") {
             get {
+                webhookLogger.info(call.parameters.toString())
```

Comment:
```suggestion
                webhookLogger.info { "${call.parameters}" }
```

do it like this so it only calls toString if info is enabled

Reply:
I forgot to remove those debug messages